content/concepts/targets.md

```diff
@@ -1,10 +1,80 @@
 ---
 title: Targets
 ---
-> webworker
-> node
-> async-node
-> node-webkit
-> electron
-> electron-main
-> electron-renderer
+## Overview
+
+Because JavaScript can be written for both server & browser, *webpack* offer's multiple deployment _targets_.
+
+*webpack* offer's a variety of deployment _targets_ that you can set in your webpack [configuration](./api/configuration).
+
+W> The webpack `target` property is not to be confused with the `output.target` property. For more information see [our guide](./concepts/output) on the `output` property.
+
+
+### Usage
+
+To set the `target` property, you simply set the target value in your webpack config: 
+
+```javascript
+// webpack.config.js
+
+module.exports = config;
+
+config = {
+  target: 'node' // <===== Just pass the name of the target  
+}
+```
+
+### Options
+
+The following is a list of values you can pass to the target property. 
+
+* `"web"` Compile for usage in a browser-like environment (default)
+* `"webworker"` Compile as WebWorker
+* `"node"` Compile for usage in a node.js-like environment (use `require` to load chunks)
+* `"async-node"` Compile for usage in a node.js-like environment (use `fs` and `vm` to load chunks async)
+* `"node-webkit"` Compile for usage in webkit, uses jsonp chunk loading but also supports build in node.js modules plus require("nw.gui") (experimental)
+* `"electron"` Compile for usage in [Electron](http://electron.atom.io/) – supports `require`-ing Electron-specific modules.
```

Comment:
As a sidenote, does it give a deprecation warning if you try to use the old name?